docs/PAYLOAD.md

```diff
@@ -72,6 +72,121 @@ Some ways to handle this *double* event are:
 - send two pushes, one to be processed in the background, and the other to show up in the shade.
 - include a unique ID in your push so you can check to see if you've already processed this event.
 
+# Message Format Overview
+
+## Android Message Format
+
+The JSON push message can contain the following fields, see https://developers.google.com/cloud-messaging/http-server-ref for a complete list.
+
+```javascript
+var content = {
+  "priority": "normal", // Valid values are "normal" and "high."
+  "content_available": "0", // see "Background Notifications" below
```

Comment:
No, this is not the correct location for this property for this plugin. Yes, I know this is where the above reference says it should be but it will screw up the receipt of the incoming push.

Reply:
Ah, so do you _know_ what the correct location actually is?

Reply:
below the docs say that it should be inside the "data" and named, "content-available": 0 - right?